x/pylons/module.go

```diff
@@ -196,6 +196,14 @@ func (am AppModule) EndBlock(ctx sdk.Context, _ abci.RequestEndBlock) []abci.Val
 				am.keeper.UnlockItemForExecution(ctx, item, pendingExec.Creator)
 			}
 
+			// if execution has payments attached, add them to the refund list
+			if am.keeper.HasPaymentProcessHistory(ctx, pendingExec.Id) {
+				ph, _ := am.keeper.GetPaymentProcessHistory(ctx, pendingExec.Id)
```

Comment:
this is an example of using the plus sign thingy to identify the exact line of code where there's an issue.